### PR TITLE
Correct link

### PR DESCRIPTION
**Pull request by ShalokShalom**
404 fixed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![Crates.io](https://img.shields.io/crates/v/wgpu.svg)](https://crates.io/crates/wgpu)
 [![Gitter](https://badges.gitter.im/gfx-rs/webgpu.svg)](https://gitter.im/gfx-rs/webgpu)
 
-This is an experimental [WebGPU](https://www.w3.org/community/gpu/) implementation as a native static library. It's written in Rust and is based on [gfx-hal](https://github.com/gfx-rs/gfx) and [satellite](https://github.com/gfx-rs/gfx-memory) libraries. The corresponding WebIDL specification can be found at [gpuweb project](https://github.com/gpuweb/gpuweb/blob/master/design/sketch.webidl).
+This is an experimental [WebGPU](https://www.w3.org/community/gpu/) implementation as a native static library. It's written in Rust and is based on [gfx-hal](https://github.com/gfx-rs/gfx) and [satellite](https://github.com/gfx-rs/gfx-memory) libraries. The corresponding WebIDL specification can be found at [gpuweb project](https://github.com/gpuweb/gpuweb).
```
Comment (kvark) [inline README.md]: the actual spec now lives at https://github.com/gpuweb/gpuweb/blob/master/spec/index.bs

Comment (ShalokShalom) [reply README.md]: Corrected, thanks.

**Comment by kvark** [comment]
Thank you! Please squash to proceed.

**Comment by ShalokShalom** [comment]
Thanks. I think this is an issue in the preview. When you look closely, is it the correct line number and also the editor confirms that.

**Comment by kvark** [comment]
I'm referring to the fact the PR has 2 commits, and it should be one in this case.

**Comment by ShalokShalom** [comment]
How to solve that?

**Comment by kvark** [comment]
Git rebase -i HEAD^2
Squash
Then git push -f origin HEAD

**Comment by ShalokShalom** [comment]
Oh sorry. I am only using this GUI.

**Pull request closed without merging**